PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -339,6 +340,8 @@ def miniAOD_customizeCommon(process):
         updatedTauName = _updatedTauName,
         toKeep = ['deepTau2017v2p1']
     )
+    from Configuration.Eras.Modifier_phase2_common_cff import phase2_common #Phase2 Tau MVA
+    phase2_common.toModify(tauIdEmbedder.toKeep, func=lambda t:t.append('newDMPhase2v1')) #Phase2 Tau MVA
```

Comment:
do we need to keep the 2017 tau ID for phase 2, or can we just store the new one?

Reply:
yes, we would like to keep it for comparison.